README.md

```diff
@@ -2029,7 +2041,7 @@ This is where data is stored and also where different processing takes place (e.
 <details>
 <summary>What is a master node?</summary><br><b>
 
-Par of a master node responsibilites:
+Par of a master node responsibilites: 
```

Comment:
What is the responsibility o the empty space ` ` here?

Reply:
no idea, maybe a missclick

Reply:
I fixed it

Reply:
Can you also fix "Par" to "Part"?